### DISCUSSION TITLE
I need help setting up to record my Discord/Zoom calls

### DISCUSSION THREAD

**Discussion by onselakin** [ExistentialAudio/BlackHole#801] [Q&A]
I'm trying to record my Discord/Zoom D&D gameplay calls and later turn them into text. What I need is to record both my own voice and the audio coming from Discord or Zoom. Here's my current setup.

Multi-output device:
![image](https://github.com/user-attachments/assets/0fcd83d1-d10e-4ba5-a134-6eaba01c378d)

Aggregate Device:
![image](https://github.com/user-attachments/assets/45e5209c-e1a3-4fdf-bb69-db3be8907b2c)

In my code I choose `MacBookPro Microphone` as the source. Also tried it with various options (like BlackHole 2ch and other aggregate devices that had the microphone in the device list) but, I can either record my voice or the sounds from the Multi Output device, but not both. 

What am I missing?

**Comment by devinroth** [comment]
Record from the aggregate. You’re probably just forgetting to capture channel 3 which is your mic according to your screenshot.
